tests/data_model/test_client.py

```diff
@@ -39,6 +39,7 @@ def setUp(self):
         self.mock_component_run = ComponentRun("mock_component_run")
         self.mock_component_run_dict = {
             "component_name": "mock_component_run",
+            "notes": None,
```

Comment:
"nodes" : ""

Reply:
Ah, the default in the `entities.ComponentRun.notes` is set to `None` though (the assumption being that adding notes is optional). So setting notes to  `""` in `test_client.py` will give a test  error. 

I can update the `entities.ComponentRun.notes` default to something else if that would align with the `models.py` module? 

Also added `TestAddNotes` to the `ComponentRun` test suite in the latest commit.

Reply:
Got it. Thanks for the new test! And yeah i think it's a good idea to update `entities.ComponentRun.notes` default to "" so everything is aligned :)

Reply:
Cool, updated `test_client.py` and `entities.ComponentRun.notes` accordingly